### ISSUE TITLE
Auto upgrader/downgrader

### ISSUE THREAD

**Issue by mineTomek** [modrinth/code#805]
### Is your suggested enhancement related to a problem? Please describe.

It's always kind of annoying to change each mod's version eg. from 1.20 to 1.20.1.

### Describe the solution you'd like

The perfect solution would be button that would take each mod and swap it for the newest mod version for the Minecraft version that the installation is in.

### Describe alternatives you've considered

The update button could also do the same thing. Or if the user would need to change the Minecraft version down (eg. 1.20 to 1.19.4) it could say 'downgrade'.

**Comment by Madis0** [comment]
And the mods that aren't available for said version should be disabled?

**Comment by mineTomek** [comment]
Yes, I think so.

**Comment by Theov0re** [comment]
This would be so useful I almost didn't realize this was a duplicate issue in my desperation to make this a reality. Although, I thought I should mention that perhaps this could be decided in the settings when you change the version. A question pops up asking if you want to update the mods, with agreement leading to every mod that has an applicable version having the latest version downloaded and enabled, while the rest would be disabled. Potentially you could also use the 'Update All' button to update the mods that had newly released applicable versions and enable them. Although, perhaps another button would be more useful for this purpose.

**Comment by mineTomek** [comment]
Yes, kind of like a checkmark, like the advanced options that's already there.

**Comment by Codingale** [comment]
This would be amazing, I've came from a cursed app that had this and you can upgrade easily but I think it warns about the version mismatch not sure, but I recently installed a modpack, copied over the mods so I can install more but realized one of the mods extra features is actually a mod and couldn't install it because it's for 1.20.2 and the modpack is 1.20.4 (there's versions for all them in 1.20.2 also thankfully)

If possible also during this make it to where modpacks are editable so I can easily add extra mods for more features like the sodium extra mod etc
